rt_gene/src/rt_gene/extract_landmarks_method_base.py

```diff
@@ -26,13 +26,13 @@ def __init__(self, device_id_facedetection, checkpoint_path_face=None, checkpoin
 
         tqdm.write("Using device {} for face detection.".format(device_id_facedetection))
 
+        self.use_cuda = 'cuda' in device_id_facedetection
```

Comment:
would it not be best if 
` self.device = 'cuda' if 'cuda' in device_id_facedetection else 'cpu' `

then every `nn.Module.cuda()` ends up being `nn.Module.to(self.device)`

Reply:
Sorry for being a bit slow to respond. I wanted to kick off a compute run before I came back to this, and it took a while to put everything together.

This is my first time using torch, so I'm working things out as I go.

`nn.Module.to(device)` accepts a `torch.device` ( https://pytorch.org/docs/stable/tensor_attributes.html#torch.torch.device ). It sounds like it can either be of the form `cuda:0` or just `cuda`, which causes it to pick the "current cuda device".

a) Would it work to `--device-id-facedetection` default to `cuda` rather than `cuda:0`?
b) Would it work to pass `device` down without stripping the `:0`?

I'm heading to bed now (UK time), but I have all of next week off, so I should be able to devote more to time to this then.

Reply:
I have gone with solution b) because I *think* that if --device-id-facedetection is specified as `cuda:0` then `.to('cuda:0')` should be the same as `.to('cuda')`. I don't even have a single CUDA enabled device, so have no idea how to test this with 2 CUDA devices :-D.

Reply:
Sorry for being late - work is going mental. 
I don't have two cuda devices either but I think that the correct thing to is option B. It might even be better to have a `device_id_gazeestimation` parameter and then all neural networks can optionally run on different cuda cores but that's not the point of this pull_request. 

Thank you for making this pull request, it's great.